tests/Mock/FetchTransactionRequestPaid.txt

```diff
@@ -0,0 +1,111 @@
+HTTP/1.1 200 OK
+Date: Wed, 26 Sep 2018 15:46:18 GMT;
+Server: Apache;
+Expires: Mon, 26 Jul 1990 05:00:00 GMT;
+Last-Modified: Wed, 26 Sep 2018 15:46:18 GMT;
+Cache-Control: post-check=0, pre-check=0;
+Pragma: no-cache;
+X-Frame-Options: SAMEORIGIN;
+X-XSS-Protection: 1; mode=block;
+X-Content-Type-Options: nosniff;
+Connection: close;
+Content-Type: application/json;
+
+{
+	"id": "672dee93-5223-8f13-1f71-771807002314",
+	"serviceId": "SL-7718-0700",
```

Comment:
This is ours right? (because we proabaly PR-back into public repo, remove this one?

Reply:
yes it can be public true

Reply:
will remove, indeed public but not neat to have it here